packages/nx/src/utils/package-manager.ts

```diff
@@ -26,6 +26,7 @@ export interface PackageManagerCommands {
   exec: string;
   dlx: string;
   list: string;
+  pack: string;
```

Comment:
This has been added but i'm not sure if a better name can be used. bun uses bun install at top level and bunx but rest of its package manager commands is at bun pm level